hack/common/consumer.sh

```diff
@@ -53,8 +53,8 @@ spec:
           imagePullPolicy: Always
           args:
             - "--local-api-addr=127.0.0.1:9089"
```

Comment:
this should change to consumer-events-subscription-service.${NamespaceConsumerTesting}.svc.cluster.local:9043

Reply:
since this is the endpoint URI that producer API need to reach and it is in a different pod than the producer, so can not use local host.

Reply:
Who will set `NamespaceConsumerTesting`, and where will it be set?

Reply:
it is defined here https://github.com/redhat-cne/cloud-event-proxy/blob/main/hack/common/common.sh#L6

Reply:
Namespace `cloud-events` is for consumer example manifests. In the e2e test it uses a different namespace.